modules/standard/FileSystem.chpl

```diff
@@ -988,7 +988,7 @@ iter glob(pattern: string = "*", followThis, param tag: iterKind): string
   var glb : glob_t;
   if (followThis.size != 1) then
     compilerError("glob() iterator can only be zipped with 1D iterators");
-  var r = followThis(1);
+  var r = followThis(0);
```

Comment:
For most of our rectangular/regular data types, including 1D iterators, `followThis` is a tuple of 0-based ranges.  This is saying "We just want the initial element of the tuple [since we are a 1D object]" and that initial element has changed from being index 1 of the tuple to index 0 of the tuple.